pg/scripts/init.sql

```diff
@@ -52,61 +49,62 @@ create table reactions (
     sabio_rk text
 );
 
+create table organisms (
+    kegg text primary key not null,
+    entry text,
+    name text,
+    taxa text
+);
+
+create table domains (
+    abbreviation text primary key not null,
+    name text
+);
 
+create table genes (
+    kegg text,
+    ncbigene text,
+    uniprot text
+);
 
 -- import data
 copy enzymes from program 'cat /input_data/enzymes/*.txt' delimiter E'\t';
 
 copy compounds from '/input_data/supplementary/compound.txt' delimiter E'\t' CSV HEADER;
 copy ec from '/input_data/supplementary/ec.txt' quote E'\u0007' delimiter E'\t' CSV HEADER;
 copy reactions from '/input_data/supplementary/reaction.txt' delimiter E'\t' CSV HEADER;
-
-
+copy organisms from '/input_data/supplementary/organism.txt' delimiter E'\t' CSV HEADER;
+copy domains from '/input_data/supplementary/domain.txt' delimiter E'\t' CSV HEADER;
+copy genes from program 'cat /input_data/supplementary/gene/*.txt' delimiter E'\t';
 
 -- create lookup indexes
-create index on enzymes("gene");
-create index on enzymes("organism");
-create index on enzymes("domain");
-create index on enzymes("reaction_id");
-create index on enzymes("compound");
+create index on reactions using gist (name, kegg);
+create index on compounds using gist (kegg);
+create index on compounds using gin (string_to_array(name, ';'));
+create index on ec using gist (name, ec);
+create index on organisms (kegg);
+create index on organisms using gist (name);
+create index on domains (abbreviation);
```

Comment:
Is this needed since it's a primary key? I noticed that the other similar indexes have been removed, which is correct I think since these are indexed by default.

Reply:
I haven't checked if it's _needed_, just intuitively thought it would faster if indexed.

Reply:
I think that's a valid assumption, but then I don't understand why the other ones are removed.

Reply:
Sorry for not being able to follow, I'm not sure what specifically you're thinking of as removed?

For more context, the main change here is indexing on the separate tables rather than the materializes views of the `enzymes` table.

Reply:
Ah sorry! I was looking at the `enzymes` lines that were removed and thought they were for manually creating the primary index.

Reply:
Sorry again. Unresolving this because I forgot to ask about why line 87 `create index on domains (abbreviation);` does not use the gist index.

Reply:
Th `domain abbreviation` is a single capital letter, so I thought a gist index would not be necessary.

Reply:
I thought the index of the `abbreviation` might be relevant when navigating to routes such as `/gotenzymes/domain/E`, but I have nothing against removing it.

Reply:
A primary key automatically gets a `btree` index so this would just be duplicating it. That was a main source of confusion for me before as well. It has been removed in https://github.com/MetabolicAtlas/MetabolicAtlas/pull/1062/commits/8547e6ebf003bd817bf407c1696299f931ade869